File: ios/RNMBX/RNMBXCustomLocationProviderViewManager.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_REMAP_MODULE(RNMBXCustomLocationProvider, RNMBXCustomLocationProviderViewManager, RCTViewManager)

// circle layer props
RCT_EXPORT_VIEW_PROPERTY(coordinate, NSArray)
RCT_EXPORT_VIEW_PROPERTY(heading, NSNumber)

@end

File: ios/RNMBX/RNMBXCustomLocationProviderVIewManager.swift
@objc(RNMBXCustomLocationProviderViewManager)
class RNMBXCustomLocationProviderViewManager: RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
  
    override func view() -> UIView! {
      let layer = RNMBXCustomLocationProvider()
      //layer.bridge = self.bridge
      return layer
    }
}
